addons/bindings.md

```diff
@@ -6,9 +6,11 @@ layout: documentation
 
 # List of Available Bindings
 
-{% assign bindings = site.data.bindings | sort: 'label.toLowerCase()' %}
+{% assign bindings = "" | split: "|" %}
```

Comment:
["You cannot initialize arrays using only Liquid. You can, however, use the split filter to break a single string into an array of substrings."](https://help.shopify.com/themes/liquid/basics/types#initializing-arrays).  Some example (I forget where) showed this as a workaround, and my local testing didn't show an empty array element.